src/helpers/FormioLoader/FormioLoader.js

```diff
@@ -23,7 +23,7 @@ const requestPluginHandler = (requestArgs, opts) => {
   // the request url generated by recaptcha component doesn't honor the form options
   // https://github.com/formio/formio.js/blob/master/src/components/recaptcha/ReCaptcha.js#L114
   // hence the workaround is to inject the project id to the url
-  if (requestArgs.url.includes(`${opts.formio.base}/recaptcha`)) {
+  if (requestArgs?.url?.includes(`${opts.formio.base}/recaptcha`)) {
```

Comment:
what does ? do to this code?

Reply:
check for object and then look for properties

Reply:
shorthand for 
`if (requestArgs && requestArgs.url && requestArgs.url.includes(`${opts.formio.base}/recaptcha`))`